src/micrate.cr

```diff
@@ -197,8 +197,9 @@ module Micrate
   end
 
   def self.logger
-    @@logger ||= Logger.new(STDOUT).tap do |l|
-      l.level = Logger::UNKNOWN
+    @@logger ||= Log.for("").tap do |l|
```

Comment:
should we provide a name? `Log.for("micrate")`

Reply:
I'll change that right now :smiley: